### ISSUE TITLE
64 bit Arm support  (Debian) 

### ISSUE THREAD

**Issue by Ganzhans** [gotson/komga#1043]
### Steps to reproduce

running of 32bit arm (raspberry PI ) before  now 64 Bit Debian 

step 1) start the container  first steps are ok, db  exist  after a clean install
step 2) Komga container hangs up RBP  
step 3) stopping the container or reboot the RBP (without starting Komga 

### Expected behavior

docker-compose up  

without -d to see al te status lines

### Actual behavior

last status is  "Adding welcome page: class path resource [public/index.html]"    
then >> Hang up RPB

### Komga version

latest 

### Operating system

Debian arm 64 bit

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/faq/).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Please provide more details:
- `docker info`
- `uname -m`
- what's your hardware model exactly?
- Komga logs
- docker daemon logs

**Comment by Ganzhans** [comment]
My device is an Rasberry PI 3b. with 1 gig of ram  

It works but ..... ik have to stop alle running containers before starting Komga. 
but after starting it takes 760 MB of ram to run this container., is this normal it take that much ?

i have also running Pihole/Mysql/NGP/Wireguard/phpadmin/piwigo and al containers + OS is using 460 Mb of ram
so it is to much for this pi :(

maybe it's nice  in de faq/readme to list the system requirements ?

**Comment by gotson** [comment]
> with 1 gig of ram

That's going to be tough.



> but after starting it takes 760 MB of ram to run this container., is this normal it take that much ?

See https://komga.org/faq/#the-memory-consumption-is-huge

> maybe it's nice Ito add in the faq/readme to list the system requirements ?

Some people run it on raspberry pi, but to be frank "servers" with less ram than any smartphone that came out in the last 5y should probably not run multiple services.

**Comment by Ganzhans** [comment]
you have a good point !! but the containers that I use now runs perfect, but they have no big load !
it for home use !
but I stop for now, for better hardware to run this container :)
